### PR TITLE
chore(demo): add example in `hosted-dropdown`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

image

### CODE REVIEW

File: projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html [hunk 1]
```diff
@@ -23,4 +23,36 @@
         </ng-template>
     </tui-hosted-dropdown>
     <button tuiTab>Another tab</button>
+    <button
+        tuiTab
+        style="--tui-tab-margin: 5px"
```
Comment (waterplea) [inline projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html:28]: Why do we need this?

File: projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html [hunk 2]
```diff
@@ -23,4 +23,36 @@
...
+    >
+        <tui-hosted-dropdown
+            tuiDropdownAlign="right"
+            tuiDropdownHover
+            [content]="settingsDropdown"
+            [open]="!!(settingsDropdownOpened$ | async)"
```
Comment (waterplea) [inline projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html:34]: Why not simple `[(open)]` binding?

**Review by waterplea — COMMENTED** [review]
Remind me what is the purpose of this example? )

**Comment by splincode** [comment]
`Remind me what is the purpose of this example? )`

For detect bug from #2167

File: projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html]: fixed

File: projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/hosted-dropdown/examples/4/index.html]: fixed

**Comment by splincode** [comment]
@waterplea @vladimirpotekhin hello, please review me)

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**